boards/microchip/mtl_template_mec172xlj.h

```diff
@@ -0,0 +1,200 @@
+#define EC_SMI				EC_GPIO_207
+#define PM_SLP_S0_CS			EC_GPIO_221
+#define SLATE_MODE			EC_GPIO_222
+
+#define DNX_FORCE_RELOAD_EC		EC_GPIO_226
+
+#define PM_BATLOW			EC_DUMMY_GPIO_HIGH
+
+
+#define PM_DS3				EC_DUMMY_GPIO_LOW
+#define EC_S0IX_ENTRY_REQ		EC_GPIO_240
+#define WAKE_CLK			EC_GPIO_241
+#define VOL_UP				EC_GPIO_242
+#define TOP_SWAP_OVERRIDE_GPIO		EC_GPIO_244
+#define VOL_DOWN			EC_GPIO_246
+#define PM_PWRBTN_ERB			EC_GPIO_047
+#define PM_PWRBTN_CRB			EC_GPIO_165
+#define PM_PWRBTN			((platformskutype == PLATFORM_MTL_S_ERB_SKUs) ? \
+					 PM_PWRBTN_ERB : \
+					 PM_PWRBTN_CRB)
+
+#define EC_M_2_SSD_PLN			EC_GPIO_254
+#define EC_S0IX_ENTRY_ACK		EC_GPIO_255
+
+#define KBC_NUM_LOCK_ERB		EC_GPIO_255
+#define KBC_NUM_LOCK_CRB		EC_GPIO_161
+#define KBC_NUM_LOCK			((platformskutype == PLATFORM_MTL_S_ERB_SKUs) ? \
+					 KBC_NUM_LOCK_ERB : \
+					 KBC_NUM_LOCK_CRB)
+
+#define KBD_BKLT_CTRL			EC_DUMMY_GPIO_HIGH
+#define STD_ADP_PRSNT			EC_DUMMY_GPIO_LOW
+#define EC_PG3_EXIT			EC_DUMMY_GPIO_LOW
+#define VIRTUAL_BAT			EC_DUMMY_GPIO_LOW
+#define VIRTUAL_DOCK			EC_DUMMY_GPIO_LOW
+#define HOME_BUTTON			EC_DUMMY_GPIO_HIGH
+#define SMC_LID				EC_DUMMY_GPIO_HIGH
+#define DG2_PRESENT			EC_DUMMY_GPIO_LOW
+#define PEG_RTD3_COLD_MOD_SW_R		EC_DUMMY_GPIO_LOW
+#define EC_PWRBTN_LED			EC_DUMMY_GPIO_LOW
+
+/* IO expander HW strap pins IO expander 1 */
+#define SPD_PRSNT			EC_GPIO_PORT_PIN(EC_EXP_PORT_1, 0x03)
+#define DISPLAY_ID_0			EC_GPIO_PORT_PIN(EC_EXP_PORT_1, 0x04)
+#define DISPLAY_ID_1			EC_GPIO_PORT_PIN(EC_EXP_PORT_1, 0x05)
+#define DISPLAY_ID_2			EC_GPIO_PORT_PIN(EC_EXP_PORT_1, 0x06)
+#define DISPLAY_ID_3			EC_GPIO_PORT_PIN(EC_EXP_PORT_1, 0x07)
+
+/* IO expander HW strap pins IO expander 2 */
+#define THERM_STRAP			EC_GPIO_PORT_PIN(EC_EXP_PORT_2, 0x08)
+#define G3_SAF_DETECT			EC_GPIO_PORT_PIN(EC_EXP_PORT_2, 0x09)
+/* Pin2 is named as PMC_GPIO_BOOT_HALT */
+#define TIMEOUT_DISABLE			EC_GPIO_PORT_PIN(EC_EXP_PORT_2, 0x0A)
+#define ESPI_TESTCRD_DET		EC_GPIO_PORT_PIN(EC_EXP_PORT_2, 0x0B)
+#define MEM_OC_N			EC_GPIO_PORT_PIN(EC_EXP_PORT_2, 0x0C)
+/* Pin5 is named as VAL_PECI_ESPI */
+#define PECI_OVER_ESPI			EC_GPIO_PORT_PIN(EC_EXP_PORT_2, 0x0D)
+
+/* Device instance names */
+#define I2C_BUS_0			DT_NODELABEL(i2c_smb_0)
+#define I2C_BUS_1			DT_NODELABEL(i2c_smb_1)
+/* LJ-specific I2C bus */
+#define I2C_BUS_2			DT_NODELABEL(i2c_smb_3)
+#define PS2_KEYBOARD			DT_NODELABEL(ps2_0)
+#define PS2_MOUSE			DT_NODELABEL(ps2_0)
+#define ESPI_0				DT_NODELABEL(espi0)
+#define ESPI_SAF_0			DT_NODELABEL(espi_saf0)
+#define SPI_0				DT_NODELABEL(spi0)
+#define ADC_CH_BASE			DT_NODELABEL(adc0)
+#define PECI_0_INST			DT_NODELABEL(peci0)
+#define WDT_0				DT_NODELABEL(wdog)
+
+/* Button/Switch Initial positions */
+#define PWR_BTN_INIT_POS		1
+#define VOL_UP_INIT_POS			1
+#define VOL_DN_INIT_POS			1
+#define LID_INIT_POS			1
+#define HOME_INIT_POS			1
+#define VIRTUAL_BAT_INIT_POS		1
+#define VIRTUAL_DOCK_INIT_POS		1
+
+#define TIPD_PORT_0_I2C_ADDR	0x23U
```

Comment:
as no Typec in opensource, i think these can be removed

Reply:
Note: These files are just a copy of MTL-S board which has these definitions already in this repo. 
Since the USB-C module is not part, these are harmless. 
Anyway removed to avoid confusion